src/netpuncher/C4PuncherPacket.h

```diff
@@ -29,6 +29,13 @@ enum C4NetpuncherPacketType {
 
 typedef uint32_t C4NetpuncherID_t;
 
+struct C4NetpuncherID {
```

Comment:
I'm not too happy with having two distinct types `C4NetpuncherID_t` and `C4NetpuncherID`